sla_per_user_system_test.py

```diff
@@ -0,0 +1,501 @@
+                               'role': Role(session=session, name='role%d' % share),
+                               'service_level': ServiceLevel(session=session, name='sla%d' % share,
+                                                             service_shares=share)})
+
+        expected_shares_ratio = self.calculate_metrix_ratio_per_user(two_users_list=read_users)
+
+        # Create Service Levels/Roles/Users
+        self.create_auths(entities_list_of_dict=read_users)
+
+        stress_duration = 10  # minutes
+        read_cmds = [self.define_read_cassandra_stress_command(user=read_users[0], load_type=load,
+                                                               workload_type=self.WORKLOAD_THROUGHPUT, threads=250,
+                                                               stress_duration_min=stress_duration),
+                     self.define_read_cassandra_stress_command(user=read_users[1], load_type=load,
+                                                               workload_type=self.WORKLOAD_THROUGHPUT, threads=250,
+                                                               stress_duration_min=stress_duration)
+                     ]
+
+        try:
+            start_time = time.time()
+
+            read_queue = self.run_stress_and_verify_threads(params={'stress_cmd': read_cmds, 'round_robin': True})
+
+            results = self.get_operates_per_second_rates_from_c_s(read_queue=read_queue, users=read_users)
+            self.validate_if_scylla_load_high_enough(start_time=start_time,
+                                                     wait_cpu_utilization=self.MIN_CPU_UTILIZATION)
+            end_time = time.time()
+
+            self.validate_scheduler_runtime(start_time=start_time, end_time=end_time,
+                                            read_users=read_users, expected_ratio=expected_shares_ratio)
+
+            self.assertTrue(results, msg='Not received cassandra-stress results')
+
+            self.log.debug('Validate cassandra-stress ops deviation')
+            actual_shares_ratio = self.calculate_metrix_ratio_per_user(two_users_list=read_users, metrix=results)
+            self.validate_deviation(expected_ratio=expected_shares_ratio,
+                                    actual_ratio=actual_shares_ratio, msg='Validate cassandra-stress ops.')
+
+        finally:
+            self.clean_auth(entities_list_of_dict=read_users)
+
+    def test_throughput_latency_mixed_load(self):
+        """
+        Test when one user run load with high latency and another  - with high througput
+        The load is run on the full data set (that is read from both the cache and the disk)
+        Troughput - latency test:
+        - Add SLA and grant to user (before any load)
+        - user190 with 190 shares
+        - user950 qith 950 shares
+        - Each user runs load from own loader (round robin):
+           - user190 runs load with throttle
+           - user950 runs load with high throughput
```

Comment:
Expected result?

Reply:
@juliayakovlev ?

Reply:
Added TODO

Reply:
I think there is a mistake in the description per user + missing expected result.